### PR TITLE
Fix colorama windows

### PR DESCRIPTION
**Pull request by NrdyBhu1**
In Windows, sometimes the Colorama module works properly, but the powershell/cmd doesn't accept the ascii color codes.

Without os.system("title ...")
Without

With:
With

### CODE REVIEW

**Comment by NrdyBhu1** [comment]
This is what I meant

**Comment by kevinjosethomas** [comment]
This isn't the best way to implement Cross-OS support for Colorama.

According to the [Colorama's documentation](https://pypi.org/project/colorama/), you need to run ``colorama.init()`` to make it work on Windows devices too.

> ANSI escape character sequences have long been used to produce colored terminal text and cursor positioning on Unix and Macs. Colorama makes this work on Windows, too, by wrapping stdout, stripping ANSI sequences it finds (which would appear as gobbledygook in the output), and converting them into the appropriate win32 calls to modify the state of the terminal. On other platforms, Colorama does nothing.

> On Windows, calling init() will filter ANSI escape sequences out of any text sent to stdout or stderr, and replace them with equivalent Win32 calls.

> On other platforms, calling init() has no effect (unless you request other optional functionality; see “Init Keyword Args”, below). By design, this permits applications to call init() unconditionally on all platforms, after which ANSI output should just work.

> To stop using Colorama before your program exits, simply call deinit(). This will restore stdout and stderr to their original values, so that Colorama is disabled. To resume using Colorama again, call reinit(); it is cheaper than calling init() again (but does the same thing).

If you're still interested in pursuing this pull request, feel free to implement the same with coloram's init function instead of adding a title to the window

**Comment by NrdyBhu1** [comment]
oh, ok will work on that too

**Comment by NrdyBhu1** [comment]
@TrustedMercury nothing much expect, just called init, now it works perfectly

File: pymon/main.py [hunk 1]
```diff
@@ -3,11 +3,12 @@
 import time
 import argparse
 import subprocess
-from colorama import Fore, Style
+from colorama import Fore, Style, init
 from watchdog.observers import Observer
 from watchdog.events import PatternMatchingEventHandler
 
 def main():
+    init()
```
Comment (kevinjosethomas) [inline pymon/main.py]: Move init() after the docstring

File: pymon/main.py [hunk 2]
```diff
@@ -30,7 +31,7 @@ def main():
     def handle_event(event):
 
         global process
-
+
```
Comment (kevinjosethomas) [inline pymon/main.py]: Unnecessary spaces added

File: pymon/main.py [hunk 3]
```diff
@@ -46,7 +47,7 @@ def handle_event(event):
     observer.schedule(event_handler, os.getcwd(), recursive=True)
 
     observer.start()
-
+
```
Comment (kevinjosethomas) [inline pymon/main.py:48]: More unnecessary spaces

**Review by kevinjosethomas — CHANGES_REQUESTED** [review]

File: pymon/main.py (hunk 1 above)
Comment (NrdyBhu1) [reply pymon/main.py]: ok sure

File: pymon/main.py (hunk 3 above)
Comment (NrdyBhu1) [reply pymon/main.py:48]: Removed extra spaces.
Request and review more changes if needed

**Pull request merged**